packages/ui/src/helpers/index.ts

```diff
@@ -70,6 +70,13 @@ export function enumToMultiSelectList<T>(items: readonly T[], formatter?: (val:
   return items.map((i) => ({ id: i, label: formatter ? formatter(i) : `${i}` }));
 }
 
+export function convertToMultiSelectOption(id: string, label?: string, shortLabel?: string) {
+  if (!id) {
```

Comment:
why do we do this?

Reply:
Because if ID is null then it'll just return the options as undefined

Reply:
In fairness I think this can be improved by making it generic, just not had the time to look at it again

Reply:
But how can I'd be null?